ch_6_GPU_Acceleration/environment_setup.md

```diff
@@ -95,10 +106,167 @@ sudo apt install -y intel-oneapi-common-vars=2023.2.0-49462 \
+wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torchvision-0.15.2a0%2Bcxx11.abi-cp39-cp39-linux_x86_64.whl
+wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/intel_extension_for_pytorch-2.0.110%2Bxpu-cp39-cp39-linux_x86_64.whl
+```
+
+Then, install dependencies directly from the archived wheels and install `bigdl-llm`:
+```bash
+# install the packages from the wheels
+pip install torch-2.0.1a0+cxx11.abi-cp39-cp39-linux_x86_64.whl
+pip install torchvision-0.15.2a0+cxx11.abi-cp39-cp39-linux_x86_64.whl
+pip install intel_extension_for_pytorch-2.0.110+xpu-cp39-cp39-linux_x86_64.whl
+
+# install bigdl-llm for Intel GPU
+pip install --pre --upgrade bigdl-llm[xpu]
+```
+
+### 5. Runtime Configuration
+To use GPU acceleration on Linux, several environment variables are required or recommended before running a GPU example.
+
+For Intel Arc™ A-Series Graphics and Intel Data Center GPU Flex Series, we recommend:
+```bash
+# configures OneAPI environment variables
+source /opt/intel/oneapi/setvars.sh
+
+export USE_XETLA=OFF
+export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
+```
+
+For Intel Data Center GPU Max Series, we recommend:
+```bash
+# configures OneAPI environment variables
+source /opt/intel/oneapi/setvars.sh
+
+export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libtcmalloc.so
+export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
+export ENABLE_SDP_FUSION=1
+```
+Please note that `libtcmalloc.so` can be installed by `conda install -c conda-forge -y gperftools=2.10`
+
+
+## Windows
+
+Prerequisites: BigDL-LLM on Windows supports Intel iGPU and dGPU.
+
+Importance: Currently Windows only supports IPEX 2.1, both the following GPU driver and oneAPI installation is only compatible with **PyTorch 2.1**. In addition, we recommend using Python 3.9 since `bigdl-llm` has been tested with this version.
+
+### 1. Install Visual Studio 2022
+Install [Visual Studio 2022 Community Edition](https://visualstudio.microsoft.com/downloads/) and select “Desktop development with C++” workload
+
+
+### 2. Install GPU driver
+Install or update to the latest driver, you can download it from [this page](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html).
+
+### 3. Install Intel® oneAPI Base Toolkit 2024.0
+
+Install from [this page](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html). Choose the Windows operating system and your favored distribution, then you can download the installer(Estimated 2.7GB for the offline one and 17MB for the online one).
+
+
+### 4. Install BigDL-LLM and IPEX
+
+We recommend using [miniconda](https://docs.conda.io/projects/miniconda/en/latest/) to create a python 3.9 environment:
+
+#### Install BigDL-LLM From PyPI
+
+```bash
+conda create -n llm python=3.9 libuv
+conda activate llm
+
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
```

Comment:
I feel it maybe better to add comment here to emphasize `[xpu]` installation option on Windows corresponds to ipex2.1.